packages/universal/debug/src/setup.ts

```diff
@@ -2,18 +2,13 @@ import { defineDebug } from "@starbeam/reactive";
 
 import debug from "./debug.js";
 
-if (
-  (globalThis.Buffer as BufferConstructor | undefined) === undefined &&
-  typeof require === "function"
-) {
-  try {
-    // this is for CJS only, so require is the only option here
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const buffer = require("buffer") as { Buffer: BufferConstructor };
-    globalThis.Buffer = buffer.Buffer;
-  } catch {
-    // ignore
+if (import.meta.env.DEV) {
+  if (!import.meta.env.CJS) {
```

Comment:
`Buffer` exists globally in node environments, so we don't need to include this `Buffer`-defining code for CJS.

_I don't know what to do if CJS is used in the browser tho_